dpareto/data/adult/downloader.py

```diff
@@ -0,0 +1,43 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import pickle
+import mxnet as mx
+from mxnet import nd
+import urllib.request
+import shutil
+import ssl
+
+print("Downloading...")
+ssl._create_default_https_context = ssl._create_unverified_context
```

Comment:
what does this line do?

Reply:
Certificate verification was failing, so this is a quick way to get around that to download the files. This can be made more secure with some extra effort, if we need it to be.